test/controllers/password_resets_controller_test.rb

```diff
@@ -0,0 +1,44 @@
+require "test_helper"
+
+class PasswordResetsControllerTest < ActionDispatch::IntegrationTest
+  include ActionDispatch::TestProcess::FixtureFile
+
+  setup do
+    @person = people(:keith_registered)
+    users(:keith)
+    credentials(:keith_password)
```

Comment:
In the setup for some of these test files you are calling a fixture (like these two lines) but not saving the value. Did you mean to save the value? If not, I don’t think that it does anything so you can remove the fixture calls that don’t do an assignment.